strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/metadata/MavenMetadataManager.java

```diff
@@ -130,9 +130,13 @@ public void storeMetadata(final RepositoryPath metadataBasePath,
                      {
                          Path metadataPath = MetadataHelper.getMetadataPath(metadataBasePath, version, metadataType);
                          try (OutputStream os = new MultipleDigestOutputStream(metadataPath,
-                                                                               Files.newOutputStream(metadataPath,
-                                                                                                     StandardOpenOption.CREATE,
-                                                                                                     StandardOpenOption.TRUNCATE_EXISTING)))
+                                 Files.newOutputStream(metadataPath,
+                                         StandardOpenOption.CREATE,
+                                         StandardOpenOption.TRUNCATE_EXISTING),
+                                 metadataBasePath.getFileSystem()
+                                         .getDigestAlgorithmSet()
+                                         .toArray(new String[0]),
+                                 true))
```

Comment:
Cosmetic: Code style.

```suggestion
                                                                               Files.newOutputStream(metadataPath,
                                                                                                     StandardOpenOption.CREATE,
                                                                                                     StandardOpenOption.TRUNCATE_EXISTING),
                                                                               metadataBasePath.getFileSystem().getDigestAlgorithmSet().toArray(new String[0]),
                                                                               true))
```